adding latitude and longitude range for the map

mrastgoo opened the pull request.

glemaitre committed:
MAINT add a scheduled job for updating website

Commit:
adding latitude and longtitude range for the map

Commit:
adding latitude and longtitude range for the map

pushing the longitude more towards the right

longitude start from -165

ranges of latitude and lonitude

Commit:
pushing the longitude more towards the right

GaelVaroquaux commented:
Thanks!!

What would be really useful is that, when the user zooms, the area used by the map expands more vertically. Right now, it is not using well the screen real estate.

I don't know how to do this, by the way.

glemaitre committed:
add explanation in README

mrastgoo commented:
ok, I will look into it, see if there is a way

GaelVaroquaux committed:
Merge pull request #70 from glemaitre/is/56

MAINT add a scheduled job for updating website

GaelVaroquaux committed:
Fix cron

GaelVaroquaux committed:
Fix cron (take 2)

emmanuelle committed:
move legend

ashwinpn committed:
Resolved errors in text_block, linked to Oxford data

ashwinpn committed:
Update text_block.md

ashwinpn committed:
Update README.md

emmanuelle committed:
annotations

GaelVaroquaux committed:
Merge pull request #73 from covid19-dash/legend-toggle

move legend

GaelVaroquaux committed:
Merge pull request #74 from ashwinpn/master

Resolved errors in text_block, added link to Oxford University analysis page

GaelVaroquaux committed:
Formatting

GaelVaroquaux committed:
Style

ashwinpn commented:
@GaelVaroquaux It would a really good addition, if we were to connect the _zooming in_  with spatial granuarity (#58)  - Initially, we would have countries represented on the plot, then as we start zooming in, we go in the order countries -> states -> provinces/boroughs -> cities -> neighbourhoods.

Currently, information on provinces/boroughs and cities won't be of much use (sparse data, still early stages to derive valuable insights) - but a couple of months from now, it would be, as the virus spreads.

Obviously this would be a complex implementation, and should be thought about when all the basic parts are working well and integrated together, but I am mentioning it here to make a note of it.

ashwinpn committed:
Aligned "drag handle..." to center

The text which describes the time window slider is not aligned properly - aligning to the center makes it look better.

ashwinpn committed:
aligned to the right.

GaelVaroquaux committed:
Merge pull request #75 from ashwinpn/patch-1

Aligned "drag handles..." to center

Commit:
Merge remote-tracking branch 'origin/is/48' into is/48

mrastgoo commented:
I couldn't find any solution other than using `yaxis.scaleanchor` and `yaxis.scaleratio`  of the `layout` [check this ref](https://community.plot.ly/t/height-aspect-ratio/6711/2), however I dont see any changes for the `plotly.express.choropleth` object. 
@emmanuelle do you have any suggestion ?
